app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,7 @@
-// = require_tree .
-// = require_self
-// = require select2
-// = require select2-bootstrap
```

Comment:
Dilwoar seems to think we'll still need the last two lines above ^, e.g. `@import "select2".

Doing so breaks my CSS, and not doing so doesn't seem to break anything 🤔

Reply:
Select2 is used in a few places - we can probably just do `@import "select2"` and `@import "select2-bootstrap"`

Reply:
Done 👍